.github/actions/k8s-compatibility-test/action.yaml

```diff
@@ -0,0 +1,46 @@
+# Runs Kubernetes version compatibility check
+# Requires as input IMG ENV variable for make target
```

Comment:
Do we need to have this as description, since it's already clearly visible in the action definition?